### PR TITLE
Lokalise: Translations update

### PR DESCRIPTION
**Pull request by carlobeltrame**
This pull request was initiated by Lokalise (user Carlo Beltrame) at 2020-11-17 18:07:29

Thanks @feigenmann for the translations

### CODE REVIEW

File: frontend/src/locales/de-CH-scout.json [hunk 1]
```diff
@@ -8,5 +15,12 @@
   },
   "global": {
     "language": "Deutsch (Pfadi)"
+  },
+  "views": {
+    "camp": {
+      "navigationCamp": {
+        "material": "Mat"
```
Comment (usu) [inline frontend/src/locales/de-CH-scout.json]: Really needed in de-Scout?

**Review by usu — APPROVED** [review]

File: frontend/src/locales/de-CH-scout.json (hunk 1 above)
Comment (carlobeltrame) [reply frontend/src/locales/de-CH-scout.json]: What do you mean by "needed"? I didn't feel like "Mat" is a correct general german translation, but it's still commonly used by the scouts. I can remove it if you think it's too informal for a web application.

**Review by pmattmann — APPROVED** [review]

File: frontend/src/locales/de-CH-scout.json (hunk 1 above)
Comment (usu) [reply frontend/src/locales/de-CH-scout.json]: The de.json value is "Mat.". Not really convinced that omitting the dot already makes it a scout slang.

Not that I have a strong opinion on that. Just a general feeling that I'd limit the Scout-specific translations to vocabulary that really is very scout specific.

Approved from my side with or without.

Comment (carlobeltrame) [reply frontend/src/locales/de-CH-scout.json]: Fair point, it should definitely be written out "Material" in German. I've also removed Mat from scout German for now.

**Pull request merged**